ui/app/styles/components/copy-button.scss

```diff
@@ -0,0 +1,16 @@
+.copy-button {
+  padding-left: 0.5rem;
+  margin-bottom: 2px;
+
```

Comment:
This might help the user know that they can click on it:
```suggestion
cursor: pointer;

```

Reply:
It actually does have that property, sadly the software I use to record GIFs causes the cursor to not update properly 😞

Reply:
Oh ok, great! Looks nice otherwise to me design-wise :+1: